### PR TITLE
GEODE-7536 - Break Gfsh out into separate module

### PR DESCRIPTION
**Pull request by yozaner1324**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by yozaner1324** [comment]
This initial PR contains two commits. The first should essentially just contain file moves. The second contains code changes that would be more relevant for reviewing.

**Comment by jdeppe-pivotal** [comment]
@jmelchio For your review

File: geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java [hunk 1]
```diff
@@ -203,8 +202,8 @@
 
     config = system.getConfig();
     try {
-      commandProcessor =
-          new OnlineCommandProcessor(system.getProperties(), cache.getSecurityService(), cache);
+      commandProcessor = cache.getService(CommandProcessor.class);
+      commandProcessor.init(cache);
```
Comment (kohlmu-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java]: Is this a redundant call. The initialization is done before the service is returned

Comment (yozaner1324) [reply geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java]: Thanks for pointing this out, it should be fixed now.

**Pull request merged**